src/test/javascript/spec/component/exam/participate/exam-participation-cover.component.spec.ts

```diff
@@ -334,4 +335,24 @@ describe('ExamParticipationCoverComponent', () => {
         component.studentExam.submitted = false;
         expect(component.studentFailedToSubmit).toBeFalse();
     });
+
+    it('should not display timer when exam was not submitted', () => {
+        jest.spyOn(component, 'studentFailedToSubmit', 'get').mockReturnValue(true);
+        component.startView = false;
+
+        fixture.detectChanges();
+
+        const examTimerDebugElement = fixture.debugElement.query(By.directive(ExamTimerComponent));
+        expect(examTimerDebugElement).toBeFalsy();
+    });
+
+    it('should display timer during working time', () => {
+        jest.spyOn(component, 'studentFailedToSubmit', 'get').mockReturnValue(false);
+        component.startView = false;
+
+        fixture.detectChanges();
+
+        const examTimerDebugElement = fixture.debugElement.query(By.directive(ExamTimerComponent)); // Replace with the actual component class name
```

Comment:
What do you want to express with the comment?

Reply:
Nothing, I just forgot to remove it. 

Thanks for the heads up!